harness/determined/deploy/gcp/cli.py

```diff
@@ -48,7 +48,11 @@ def deploy_gcp(command: str, args: argparse.Namespace) -> None:
     if not os.path.exists(args.local_state_path):
         os.makedirs(args.local_state_path)
     os.chdir(args.local_state_path)
-    print("Using local state path:", args.local_state_path)
+
+    if args.tf_state_gcs_bucket_name:
```

Comment:
`det deploy gcp down` is erroring out on this line due to no arg being parsed in the down case.